docs/install.md

```diff
@@ -105,6 +105,100 @@ grep /etc/group -e "docker"
 sudo systemctl restart coder.service
 ```
 
+## Kubernetes via Helm
+
+Before proceeding, please ensure that you have both Helm 3.5+ and the
+[latest version of Coder](https://github.com/coder/coder/releases) installed.
+You will also need to have a Kubernetes cluster running K8s 1.19+.
+
+> See our [Helm README](https://github.com/coder/coder/blob/main/helm#readme)
+> file for additional information.
+
+> ⚠️ **Warning**: Helm support is new and not yet complete. There may be changes
+> to the Helm chart between releases which require manual values updates. Please
+> file an issue if you run into any issues.
+>
+> Additionally, the Helm chart does not currently automatically configure a
+> Service Account and workspace template for use in Coder. See
+> [#3265](https://github.com/coder/coder/issues/3265).
+
+
+1. Create a PostgreSQL deployment. Coder does not manage a database server for
+   you.
+
+    - If you're in a public cloud such as
+      [Google Cloud](https://cloud.google.com/sql/docs/postgres/),
+      [AWS](https://aws.amazon.com/rds/postgresql/),
+      [Azure](https://docs.microsoft.com/en-us/azure/postgresql/), or
+      [DigitalOcean](https://www.digitalocean.com/products/managed-databases-postgresql),
+      you can use the managed PostgreSQL offerings they provide. Make sure that
+      the PostgreSQL service is running and accessible from your cluster. It
+      should be in the same network, same project, etc.
+    - You can install Postgres manually on your cluster using the
+      [Bitnami PostgreSQL Helm chart](https://github.com/bitnami/charts/tree/master/bitnami/postgresql#readme).
+      There are some
+      [helpful guides](https://phoenixnap.com/kb/postgresql-kubernetes) on the
+      internet that explain sensible configurations for this chart.
```

Comment:
Should we support specifying the Postgres URL via a Kubernetes secret, or simply rely on the cluster networking to restrict connections?

GitLab's helm chart [appears to support this](https://docs.gitlab.com/charts/advanced/external-db/index.html), and it feels quite dangerous to me to hardcode the connection string...

Reply:
I'll change the example value to use fromValue and recommend this in the docs: https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables